test/e2e/ctx/ctx.go

```diff
@@ -132,6 +132,10 @@ func (ctx TestContext) DumpNamespaceArtifacts(namespace string) error {
 		"TEST_ARTIFACTS_DIR=" + logDir,
 		"KUBECONFIG=" + kubeconfigPath,
 	}
+	kubectl := os.Getenv("KUBECTL")
+	if kubectl != "" {
+		envvars = append(envvars, fmt.Sprintf("KUBECTL=%s", kubectl))
+	}
```

Comment:
nit: can we condense this into the literal?

```go
envvars := []string{
    "TEST_NAMESPACE=" + namespace,
    "TEST_ARTIFACTS_DIR=" + logDir,
    "KUBECONFIG=" + kubeconfigPath,
    "KUBECTL=" + os.Getenv("KUBECTL"),
}
```

Related: could also avoid a lot of dependency issues by porting `collect-ci-artifacts.sh` to go?

Reply:
What happens if KUBECTL is empty (which should be the default case)?

Reply:
I imagine the script would detect an empty/non-existent environment variable and would use a default

Reply:
I'm not sure where I was going with my last comment - makes sense to me given we already provide a default value when the $KUBECTL variable is undefined or empty, so we should be good to go. Updated in the last force push.